src/test/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHourControllerTests.java

```diff
@@ -56,9 +70,18 @@
     @MockBean
     TutorAssignmentRepository mockTutorAssignmentRepository;
 
+    @MockBean
+    CourseRepository mockCourseRepository;
+
     @MockBean
     TutorRepository mockTutorRepository;
 
+    @MockBean
+    CSVToObjectService mockCSVToObjectService;
```

Comment:
This is not used in your test, we should remove this, which means we should also remove the import `import edu.ucsb.ucsbcslas.services.CSVToObjectService;`